apps/src/templates/checkbox-dropdown.module.scss

```diff
@@ -1,25 +1,54 @@
 @import "color.scss";
 
+/* stylelint-disable */
+// checkbox active background color
+$light_primary_500: #0093A4;
```

Comment:
Could you add this color to `color.scss` please, then you wouldn't need the disable lint line above it?

Reply:
That's intended, but thanks for noticing! The color is part of our new colors set that's coming soon. I've also used it in the same way in checkbox component for now. Once new colors set will be ready and on staging, these color declarations will be deleted